base_automation_team_activity/tests/test_team_activity.py

```diff
@@ -0,0 +1,109 @@
+# Copyright 2024 Binhex - Adasat Torres
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo.tests import tagged
+from odoo.tests.common import TransactionCase
+
+
+@tagged("-at_install", "post_install")
+class TestTeamActivity(TransactionCase):
+    def setUp(self):
+        super().setUp()
+
```

Comment:
```suggestion
    @classmethod
    def setUpClass(cls):
        super().setUpClass()
        cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))
```
and rework the rest w/ `cls`. 
Goal: run this code only once and not for _all_ the tests, reduce overhead due to mail tracking.

Reply:
Hello, I haven't been able to make this change because switching to the `setUpClass()` method generates an error related to the `env` attribute.

Reply:
I have reviewed the `common` file in Odoo 14.0 and noticed that `setUpClass` only exists in the `SingleTransactionCase` class, not in the `TransactionCase`. I will look into it further and update the tests when I have more time.